snooty/taxonomy.toml

```diff
@@ -4,55 +4,221 @@ name = "reference"
 [[genres]]
 name = "tutorial"
 
+# trying to map to docs "name" property 
+# for target_platforms and sub_platforms
+# as much as possible
 [[target_platforms]]
 name = "atlas"
-[[target_platforms.versions]]
-name = "v1.2"
-[[target_platforms.versions]]
-name = "master"
+
+  [[target_platforms.sub_platforms]]
+  name = "atlas-app-services"
+  display_name = "App Services"
+
+  [[target_platforms.sub_platforms]]
+  name = "charts"
+  display_name = "Charts"
+
+  [[target_platforms.sub_platforms]]
+  name = "atlas-cli"
+  display_name = "Atlas CLI"
+
+  [[target_platforms.sub_platforms]]
+  name = "data-api"
+  display_name = "Data API"
+
+  [[target_platforms.sub_platforms]]
+  name = "data-federation"
+
+  [[target_platforms.sub_platforms]]
+  name = "data-lake"
+
+  [[target_platforms.sub_platforms]]
+  name = "device-sync" #404
+
+  [[target_platforms.sub_platforms]]
+  name = "kubernetes-operator"
+
+  [[target_platforms.sub_platforms]]
+  name = "online-archive"
+
+  [[target_platforms.sub_platforms]]
+  name = "search"
+
+  [[target_platforms.sub_platforms]]
+  name = "stream-processing"
+
+  [[target_platforms.sub_platforms]]
+  name = "triggers"
+
+[[target_platforms]]
+name = "bi-connector"
+display_name = "BI Connector"
+
+[[target_platforms]]
+name = "com"
 
 [[target_platforms]]
-name = "atlas-cli"
-[[target_platforms.versions]]
-name = "v1.2"
-[[target_platforms.versions]]
-name = "master"
+name = "c2c"
+display_name = "Cluster-to-Cluster Sync"
 
 [[target_platforms]]
-name = "manual"
-[[target_platforms.versions]]
-name = "v1.0"
-[[target_platforms.versions]]
-name = "master"
+name = "community-kubernetes-operator"  # 404
 
 [[target_platforms]]
-name = "spark-connector"
-[[target_platforms.versions]]
-name = "v2.0"
-[[target_platforms.versions]]
-name = "v2.1"
+name = "compass"
 
 [[target_platforms]]
-name = "node"
-[[target_platforms.versions]]
-name = "v4.9"
+name = "database-tools"
+
+[[target_platforms]]
+name = "drivers"
+
+[[target_platforms]]
+name = "enterprise-kubernetes-operator"
+
+[[target_platforms]]
+name = "kafka-connector"
+
+[[target_platforms]]
+name = "mongodb-analyzer" # 404
+display_name = "MongoDB Analyzer"
 
 [[target_platforms]]
 name = "mongocli"
-[[target_platforms.versions]]
-name = "v1.0"
+display_name = "MongoDB CLI"
+
+[[target_platforms]]
+name = "mongocli"
+display_name = "MongoDB CLI"
 
 [[target_platforms]]
 name = "visual-studio-extension"
-[[target_platforms.versions]]
-name = "current"
+display_name = "MongoDB for VS Code"
+
+[[target_platforms]]
+name = "mongodb-shell"
+display_name = "MongoDB Shell"
+
+[[target_platforms]]
+name = "onprem" # verify
+display_name = "Ops Manager"
+
+[[target_platforms]]
+name = "realm"
+
+  [[target_platforms.sub_platforms]]
+  name = "cpp"
+  display_name = "C++ SDK"
+
+  # nestled under docs-realm 
+  [[target_platforms.sub_platforms]]
+  name = "fluter"
```

Comment:
Is the folder flutter rather than fluter?

Reply:
thank you for the catch @sarahemlin !